dapps/assets/src/components/contacts/InviteDispatcher.ts

```diff
@@ -105,7 +105,7 @@ dispatcher
           fromAlias: data.fromAlias,
           lang: data.currLang,
         },
-        runtime.web3.utils.toWei('1'),
+        runtime.web3.utils.toWei('0'),
```

Comment:
I think this caused an error in the backend the last time I tried to send 0. Did anything change there?

Reply:
Yes. The backend can now handle the sending of zero eve :)